docs/api/clojure/ndarray.md

```diff
@@ -0,0 +1,118 @@
+# NDArray API
+
+
+The ndarray api contains tensor operations similar to `numpy.ndarray`. The syntax is also similar, except for some additional calls for dealing with I/O and multiple devices.
+
+Topics:
+
+* [Create NDArray](#create-ndarray)
+* [NDArray Operations](#ndarray-operations)
+* [NDArray API Reference](http://mxnet.incubator.apache.org/api/clojure/docs/org.apache.clojure-mxnet.ndarray.html)
+
+To follow along with this documentation, you can use this namespace to with the needed requires:
+
+```clojure
+(ns docs.ndarray
+  (:require [org.apache.clojure-mxnet.ndarray :as ndarray]
+            [org.apache.clojure-mxnet.context :as context]))
+```
+
+
+## Create NDArray
+
+Create `mxnet.ndarray` as follows:
+
+```clojure
+
+(def a (ndarray/zeros [100 50])) ;;all zero arrray of dimension 100 x 50
+(def b (ndarray/ones [256 32 128 1])) ;; all one array of dimension
+(def c (ndarray/array [1 2 3 4 5 6] [2 3])) ;; array with contents of a shape 2 x 3
+```
+
+There are also ways to convert to a vec or get the shape as an object or vec
+
+```clojure
+(ndarray/->vec c) ;=> [1.0 2.0 3.0 4.0 5.0 6.0]
+(ndarray/shape c) ;=> #object[org.apache.mxnet.Shape 0x583c865 "(2,3)"]
+(ndarray/shape-vec c) ;=> [2 3]
+```
+
+
+## NDArray Operations
+
+We provide some basic ndarray operations, like arithmetic and slice operations.
```

Comment:
NDArray

Reply:
I usually try to stay away from "we" unless it's a blog or some similar discourse with multiple authors or if that's the tone of the tutorial... a journey you're doing together.